examples/with-cloud/app/api/assistant-ui-token/route.ts

```
@@ -0,0 +1,55 @@
+  return jwt.sign(
+    {
+      sub: userId,
+      iat: Date.now(),
+      exp: Date.now() + 60 * 60 * 24 * 7, // 1 week
+    },
+    process.env["JWT_SECRET"]!,
+  );
+};
+
+const getUserIdFromJwt = (token: string) => {
+  const decoded = jwt.verify(token, process.env["JWT_SECRET"]!) as JwtPayload;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)